core/pipeline.py

```
@@ -1,76 +1,155 @@
 # core/pipeline.py
-import logging
+import os
 import json
-# Set up logging configuration (this can also be configured globally)
+import logging
+from pymongo import MongoClient
+from bson import ObjectId  # Import ObjectId if needed
```

Ruff (0.8.2)

`bson.ObjectId` imported but unused

Remove unused import: `bson.ObjectId`

(F401)